docs/source/GetStartedGuide.rst

```diff
@@ -211,6 +211,75 @@ keep reading.
           positional arguments, therefore test names (in case of
           ``avocado run``)
 
+Test References
+---------------
+
+A Test Reference is a string that can be resolved into (interpreted as)
+one or more tests by the Avocado Test Resolver. It is usually a path for
+a test file and it can be followed by a `:` and then a regular expression
```

Comment:
Well this is up to the loader plugin, `avocado-vt`, `external` does not use `path` and `SIMPLE` test also does not support regexps.